src/main/java/ch/njol/skript/structures/StructFunction.java

```diff
@@ -134,4 +169,144 @@ public String toString(@Nullable Event event, boolean debug) {
 		return (local ? "local " : "") + "function";
 	}
 
+	public static class FunctionParser {
```

Comment:
is this intended to be usable by addons? if so, i think we should move it out of this class (syntax classes are likely to move around)